io_batch_import_objs.py

```diff
@@ -118,6 +118,10 @@ def execute(self, context):
                                 clamp_size = self.clamp_size_setting,
                                 validate_meshes = self.validate_setting
                                 )
+                # assign filename (without extension) to the imported object
```

Comment:
Why the for-loop within the main loop? As is, the PR updates the filename of all objects in selection multiple times (after one object gets imported). Isn't `context.object = "new-name"` enough in this case?